### PR TITLE
[CMS-1018, CMS-1005] `site:autopilot:env-sync:enable`, `site:autopilot:env-sync:disable`, `site:autopilot:destination` commands

### PR DESCRIPTION
**Pull request by jabxjab**

### CODE REVIEW

File: src/AutopilotApi/Client.php [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getSettings(string $site_id): array
+    {
+        return $this->requestApi(sprintf('sites/%s/vrt/settings', $site_id));
+    }
+
+    /**
+     * Sets autopilot frequency setting.
```
Comment (kporras07) [inline src/AutopilotApi/Client.php]: Wrong comment I guess?

File: src/Commands/FrequencySetCommand.php [hunk 2]
```diff
@@ -2,80 +2,65 @@
...
      */
-    public function autopilotFrequencySet(
-        string $site_id,
-        string $frequency,
-        array $options = ['debug' => false]
-    ): bool {
-
-        // Allow frequency to be case-insensitive for user friendliness.
-        $frequency = strtoupper($frequency);
+    public function autopilotFrequencySet(string $site_id, string $frequency = null): ?string
```
Comment (kporras07) [inline src/Commands/FrequencySetCommand.php]: You're now using the same command/function to set and get. Could you please rename the function for clarity?

**Review by kporras07 — CHANGES_REQUESTED** [review]
Looks pretty good. Just a couple of minor comments

File: src/AutopilotApi/AutopilotClientAwareTrait.php [hunk 3]
```diff
@@ -29,6 +30,10 @@ public function getClient(): Client
             return $this->autopilotClient;
         }
 
+        if (getenv('TERMINUS_IS_TESTING_ENV')) {
```
Comment (greg-1-anderson) [inline src/AutopilotApi/AutopilotClientAwareTrait.php:33]: Isn't there a better way to do dependency injection for this in tests?

Comment (jabxjab) [reply src/AutopilotApi/AutopilotClientAwareTrait.php:33]: Tried to find a better way, this is least influencing

Comment (jabxjab) [reply src/AutopilotApi/AutopilotClientAwareTrait.php:33]: This was a starting point - https://github.com/pantheon-systems/terminus-secrets-manager-plugin/blob/main/src/SecretsApi/SecretsApi.php#L65

Comment (greg-1-anderson) [reply src/AutopilotApi/AutopilotClientAwareTrait.php:33]: Are you testing the `phar`, or is this only for unit tests?

Comment (jabxjab) [reply src/AutopilotApi/AutopilotClientAwareTrait.php:33]: This is for functional tests which executes the binary (`phar`)

**Review by kporras07 — APPROVED** [review]

**Review by stovak — APPROVED** [review]
Great work. LGTM.

**Pull request merged**